csrc/dynamic_type.h

```diff
@@ -179,7 +179,7 @@ template <typename Containers, typename... Ts>
 struct DynamicType {
   using VariantType =
       typename Containers::template VariantType<DynamicType, Ts...>;
-  VariantType value_;
+  VariantType value;
```

Comment:
Generally, a struct should use `variable_name` for its data members, but classes should use `variable_name_`. See https://google.github.io/styleguide/cppguide.html#Variable_Names